Use new "create repo from template" GitHub feature (#100)
moj-analytical-services/platform_user_guidance: 06-deploying_static.Rmd

## Step-by-step guide to depolying an static web app

### Use the webapp template

> To create a new repo for the app, use the webapp template:

Reviewer: ```suggestion
To create a new repository based on the webapp template:
```

> 1. Go to the [webapp-template repo](https://github.com/moj-analytical-services/webapp-template).

Reviewer: ```suggestion
1. Go to the [webapp-template](https://github.com/moj-analytical-services/webapp-template) repository.
```

> 2. Click __Use this template__.

Reviewer: ```suggestion
2. Select __Use this template__.
```

3. Fill in the form:
    + Owner: `moj-analytical-services`

>     + Name: The name of your app, e.g., `my-app`

Reviewer: ```suggestion
    + Name: The name of your app, for example, `my-app`
```
